packages/angular/.storybook/preview.js

```diff
@@ -0,0 +1,19 @@
+import './polyfills.js';
+
+import { addDecorator, addParameters } from '@storybook/angular';
+import { withKnobs } from '@storybook/addon-knobs';
+import { CarbonG100 } from './theme';
+
+// load global styles
+require('!style-loader!css-loader!postcss-loader!sass-loader!./preview.scss');
+
+addParameters({
+  options: {
+    theme: CarbonG100,
```

Comment:
still doing the G100 here. Should change to G10.

Reply:
🤦